packages/protocol/contracts/thirdparty/lido/src/BridgeableTokens.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.20;
```

Comment:
May I know which files are copied over from other project as-is, which ones have our own modificaiton. 

If we copied files over, we need a URL to the original source file.

Reply:
`BridgeableTokens.sol`, and `BridgingManager.sol` is taken from the already existing Lido implementation. Other file like `L1Executor.sol` is completely implemented by me.